onnx/defs/parser.h

```diff
@@ -138,6 +142,10 @@ class KeyWordMap {
     map_["model_version"] = KeyWord::MODEL_VERSION;
     map_["doc_string"] = KeyWord::DOC_STRING;
     map_["metadata_props"] = KeyWord::METADATA_PROPS;
+    map_["seq"] = KeyWord::SEQ_TYPE;
```

Comment:
tiny nit: it seems that seq is the only abbreviation here for other types. Perhaps we can just use sequence here for consistency, but it's up to you.

Reply:
Thanks. I am using the abbreviations already used for all these types, in the opschema registrations. See, for example: https://github.com/onnx/onnx/blob/00738d2032ca0a39d9124b02085a90adda96d018/onnx/defs/schema.h#L668